.github/workflows/staging.yml

```diff
@@ -0,0 +1,79 @@
+name: staging-CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This pipeline should only execute on changes of the `master`. I'm haven't worked with GitHub actions yet, but I suspect this line triggers the CI on each PR on the master

Reply:
I only used it, to verify, that the pipe is successfully executed. I also removed it now.